service_linux.go

```diff
@@ -32,7 +32,7 @@ func init() {
 
 // Get all loadbalancers on this network that is currently discovered
 // on this node.
-func (n *network) connectedLoadbalancers() []*loadBalancer {
+func (n *network) connectedLoadBalancers() []*loadBalancer {
```

Comment:
confused on the following 2 methods, are they still making sense? there should be only 1 LB per network right?